pallets/loans/src/tests.rs

```diff
@@ -939,7 +939,7 @@ macro_rules! test_pool_nav {
 					assert_err!(res, Error::<MockRuntime>::LoanCeilingReached);
 
 					// payback 50 and borrow more later
-					let repay_amount = Amount::from_inner(50 * USD);
+					let repay_amount = Amount::from_inner(200 * USD);
```

Comment:
Why this change? (and it's not out of sync with the comment)

Reply:
Removed it. Forget about that. Haha. I wanted to check how we handle over-repayment. This makes the tests work again. So forget about my comment below....